### ISSUE TITLE
DAC 25 - Medium Priority WCAG Level AA - Colour Contrast [Text] 02

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#25]
<img width="282" alt="DAC_Colour_Contrast_02" src="https://user-images.githubusercontent.com/75850106/155569027-024c6b43-0f63-4b52-9f28-20f276799ef3.png">

### Issue
Low vision users may not know that the table towards the end of the site can be filtered. This is due to the contrast of the text being less than the minimum 4.5:1 ratio. These users may have difficulty reading or even seeing the text.

Foreground: #978C80 
Background: #FFFFFF
The contrast ratio is: 3.3:1 
1.4.3 Contrast (Minimum) (AA)
Pass for large text only. Fail for regular text 1.4.6 Contrast (Enhanced) (AAA)
Fail for large and regular text

<img width="596" alt="DAC 25 Current code ref" src="https://user-images.githubusercontent.com/75850106/155569151-a1ecb078-b7eb-4f10-861f-04e973a878db.png">

### Suggested Solution
We recommend that the placeholder text is set at a darker colour so that the minimum ratio of 4.5:1 is met. We also recommend that the border is set to a darker colour and reach a minimum 3:1 ratio.
